RocketChat/Rocket.Chat
# Chore: Update delete team modal to new design

## Proposed changes (including videos or screenshots)


Now the modal has only 2 steps (steps 1 and 2 were merged)
![image](https://user-images.githubusercontent.com/40830821/119414580-2e398480-bcc6-11eb-9a47-515568257974.png)



## Issue(s)


## Steps to test or reproduce

Go to a team and try to delete it. The first two modals now are merged (should display as in the screenshot)
## Further comments
